lib/linux.js

```diff
@@ -123,15 +113,39 @@ function setAvailableApps() {
 	});
 }
 
+function isCinnamon() {
+	const args = ['writable', 'org.cinnamon.desktop.background', 'picture-uri'];
+	// output = childProcess.execFile('gsettings', args, ());
+	return cp.execFile('gsettings', args)
+		.then(out => (out.toString().trim() === 'true'))
+		.catch(err => {
+			err.toString(); // do nothing, this error is expected
```

Comment:
i cant reproduce rn, but i think your linter complains about that if there's nothing caught